packages/Webkul/Admin/src/Resources/views/layouts/master.blade.php

```diff
@@ -22,6 +22,18 @@
         @stack('css')
 
         {!! view_render_event('bagisto.admin.layout.head') !!}
+
+        @php 
+
+        $paddingNavbar = 'padding-left:56px';
+        $paddingNavbarOpen = 'padding-left:200px';
+
+        if(core()->getCurrentLocale() && core()->getCurrentLocale()->direction == 'rtl') {
+            $paddingNavbar = 'padding-right:56px';
+            $paddingNavbarOpen = 'padding-right:200px';
+        }
+
+        @endphp
```

Comment:
This should be handled with scss. No, if cases for RTL.

Reply:
ok sir